db_utils.py

```diff
@@ -67,15 +67,23 @@ def connect(self):
 
             self.logger.info("Db connected")
         else:
-            pass
+            try:
+                self.cnx.ping(reconnect=True, attempts=1, delay=1)
```

Comment:
oh, I think the attempts count needs to go before the ping, probably need to repush that change